runner/android_junit_runner/java/androidx/test/internal/runner/listener/InstrumentationResultPrinter.java

```diff
@@ -45,7 +45,7 @@
 
   private static final String TAG = "InstrumentationResultPrinter";
 
-  @VisibleForTesting static final int MAX_TRACE_SIZE = 32 * 1024;
```

Comment:
I thought the binder maximum size was 1 MB.

Reply:
Any value here is risky, in any direction we go. Decreasing max means shorter stacktraces => Espresso is less useful. Increasing the max means more risk of hitting the binder max.

What if this didn't already have a value today, how would we pick a value? 65,536 bytes is super conservative, but 131,072 bytes is also very conservative. To send multiple stacktraces at the same time you'd need to run tests in parallel and you'd need them to fail and you'd need long stacktraces for each. Isn't that a bit far fetched?

Reply:
Would it be hard to impose a limit on the aggregated size rather than individuals? If the cumulative stack trace size is big in one binder transaction, then stop adding more stack traces. This way, the common usefulness will increase (developer running a single test, tying to reproduce an issue and not seeing stack currently), while other cases will have more data too, but with some stacks fully missing.

Reply:
The proposed change is targeting `InstrumentationResultPrinter`, not `OrchestrationResultPrinter`/`OrchestrationXmlTestRunListener` which the linked issues clearly shows has issues. Also `OrchestrationResultPrinter`/`OrchestrationXmlTestRunListener` do not impose any limits on the MAX_TRACE_SIZE although the javadoc states (maybe they actually should):

> ... 
> A line by line reimplementation of {@link
>  androidx.test.internal.runner.listener.InstrumentationResultPrinter}
> ...

And also they have no dependency on `InstrumentationResultPrinter` from what I can tell.